app/assets/javascripts/components/ops/tenant-component.js

```diff
@@ -0,0 +1,94 @@
+ManageIQ.angular.app.component('tenantComponent', {
+  bindings: {
+    recordId: '=?',
```

Comment:
`@?`

Since you're passing in an id, bidirectional binding doesn't make sense.